airbyte-integrations/connectors/source-sendgrid/source_sendgrid/schemas/unsubscribe_groups.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "integer"
```

Comment:
Can you update these types with "type": ["null", "integer"] or "type": ["null", "string"], etc. This will make it so that the stream doesn't fail as easily in the future